### DISCUSSION TITLE
Feature idea: Radio sampling

### DISCUSSION THREAD

**Discussion by MrRampage2321** [FundamentalFrequency/.github#29] [Ideas]
One of the things I really liked about the OP-1 was the ability to connect to the radio and record samples of the songs playing or people talking. Considering there are wifi radio stations out there I figure something like this might be very possible. Thoughts?

**Comment by pixiesue** [comment]
A software solution through wifi or bluetooth would be great!
Like i don’t know how difficult it would be to implement.
And it has to be a fully open source solution I guess.

Found thist article about an hardware Pi4 expansion for that.
https://hackaday.com/2021/09/07/an-inexpensive-fm-receiver-for-the-raspberry-pi/

And also that hack to transmit FM from a Pi4
https://github.com/markondej/fm_transmitter
